scss/reprivata-theme/custom.scss

```diff
@@ -185,3 +195,56 @@ $display-font-family: $work-sans;
 $lead-font-size: $font-size-base * 1.125;
 $lead-font-weight: 400;
 $small-font-size: 0.889rem;
+
+// Buttons + Forms
+//
+// Shared variables that are reassigned to `$input-` and `$btn-` specific variables.
+
+// scss-docs-start input-btn-variables
+
+$input-btn-padding-y: 8px;
+$input-btn-padding-x: 16px;
+$input-btn-font-family: $open-sans;
+$input-btn-font-size: $small-font-size;
+$input-btn-line-height: 14px;
+$input-btn-focus-width: 2px;
+// $input-btn-focus-color-opacity: none;
+$input-btn-focus-color: $blue;
+$input-btn-focus-blur: none;
+// $input-btn-focus-box-shadow: 0;
+
+$input-btn-padding-y-sm: 8px;
+$input-btn-padding-x-sm: 4px;
+$input-btn-font-size-sm: $input-btn-font-size;
+
+$input-btn-border-width: $border-width;
+
+$btn-padding-y: $input-btn-padding-y;
+$btn-padding-x: $input-btn-padding-x;
+$btn-font-size: $small-font-size;
+$btn-line-height: $input-btn-line-height;
+//Button border radius
+$btn-border-radius: $border-radius;
+$btn-border-radius-sm: $border-radius;
+$btn-border-radius-lg: $border-radius;
+
+$btn-focus-box-shadow: none;
+
+// Cards
+
+// scss-docs-start card-variables
+$card-spacer-y: $spacer;
+$card-subtitle-font-size: $font-size-lg;
+$card-border-radius: $border-radius;
+
+// scss-docs-start modal-variables
+$modal-inner-padding-top: $spacer;
+$modal-inner-padding-right: $spacer + 8;
```

Comment:
Best to use multiplication and division where possible to maintain ratios. If `$spacer` changes, the ratios would completely change.